src/humanize/number.py

```diff
@@ -17,6 +17,28 @@ def ordinal(value):
     anything `int()` will turn into an integer. Anything other value will have nothing
     done to it.
 
+    Examples:
+        ```pycon
+        >>> ordinal(1)
+        '1st'
+        >>> ordinal(1002)
+        '1002nd'
+        >>> ordinal(103)
+        '103rd'
+        >>> ordinal(4)
+        '4th'
+        >>> ordinal(12)
+        '12th'
+        >>> ordinal(101)
+        '101st'
+        >>> ordinal(111)
+        '111th'
+        >>> ordinal("something else")
+        'something else'
+        >>> ordinal(None) is None
+        True
+
```

Comment:
Also, is this newline needed, and those for subsequent ones?

Reply:
Yes if there isn't a space between the three backticks and the test result doctests report it as a failure for some reason.

Reply:
Fair enough!

Yes, `pycon` is for syntax highlighting on RTD.

Thanks, I think this is ready for merge!